src/software/ai/hl/stp/play/corner_kick_play_test.py

```diff
@@ -67,90 +68,63 @@ def test_corner_kick_play_bottom_left(simulated_test_runner):
         ),
     )
 
-    # Always Validation
-    # TODO- #2781 Validation
-    always_validation_sequence_set = [[]]
 
-    # Eventually Validation
+def test_corner_kick_play_bottom_left(simulated_test_runner):
+    # TODO- #2781 Validation
     # TODO- #2781 Validation
-    eventually_validation_sequence_set = [[]]
-
     simulated_test_runner.run_test(
-        eventually_validation_sequence_set=eventually_validation_sequence_set,
-        always_validation_sequence_set=always_validation_sequence_set,
+        setup=lambda ball_and_bots_pos: corner_kick_play_test_setup(
+            ball_and_bots_pos, simulated_test_runner
```

Comment:
nit: instead of passing an object with all of the parameters, it might make it slightly more readable to pass in individual fields of the parameter object directly, since the object is created here as well.